docs/instant.md

```diff
@@ -7,14 +7,46 @@
 
 A `Temporal.Instant` is a single point in time (called **"exact time"**), with a precision in nanoseconds.
 No time zone or calendar information is present.
-As such `Temporal.Instant` has no concept of days, months or even hours.
+To obtain local date/time units like year, month, day, or hour, a `Temporal.Instant` must be combined with a `Temporal.TimeZone` instance or a time zone string.
 
-For convenience of interoperability, it internally uses nanoseconds since the [Unix epoch](https://en.wikipedia.org/wiki/Unix_time) (midnight UTC on January 1, 1970).
-However, a `Temporal.Instant` can be created from any of several expressions that refer to an exact time, including an ISO 8601 string with a time zone such as `'2020-01-23T17:04:36.491865121-08:00'`.
+<!-- prettier-ignore-start -->
+```javascript
+instant = Temporal.Instant.from('2020-01-01T00:00+05:30'); // => 2019-12-31T18:30:00Z
+instant.epochNanoseconds; // => 1577817000000000000n
+
+// `Temporal.Instant` lacks properties that depend on time zone or calendar
+instant.year; // => undefined
 
-If you have a legacy `Date` instance, you can use its `toTemporalInstant()` method to convert to a `Temporal.Instant`.
+zdtTokyo = instant.toZonedDateTimeISO('Asia/Tokyo'); // => 2020-01-01T03:30:00+09:00[Asia/Tokyo]
+zdtTokyo.year; // => 2020
+zdtTokyo.toPlainDate(); // => 2020-01-01
+
+tzLA = Temporal.TimeZone.from('America/Los_Angeles');
+zdtLA = instant.toZonedDateTimeISO(tzLA); // => 2019-12-31T10:30:00-08:00[America/Los_Angeles]
+zdtLA.year; // => 2019
+zdtLA.toPlainDate(); // => 2019-12-31
+```
+<!-- prettier-ignore-end -->
 
-Since `Temporal.Instant` doesn't contain any information about time zones, a `Temporal.TimeZone` is needed in order to convert it into a `Temporal.ZonedDateTime` or `Temporal.PlainDateTime` (and from there into any of the other `Temporal` objects.)
+`Temporal.Instant` stores a count of integer nanoseconds since the [Unix epoch](https://en.wikipedia.org/wiki/Unix_time): midnight UTC on January 1, 1970.
+For interoperability with `Date` (which uses millisecond precision) or other APIs, `Temporal.Instant` also offers conversion properties and methods for seconds, milliseconds, or microseconds since epoch.
+A `Temporal.Instant` can also be created from an ISO 8601 / RFC 3339 string like `'2020-01-24T01:04Z'` or `'2020-01-23T17:04:36.491865121-08:00'`.
+
+<!-- prettier-ignore-start -->
+```javascript
+// New Year's in India
```

Comment:
Let's skip this comment, I get that it means "this is the timestamp that _would_ be midnight 01-01 in India" but I'd want to keep away from any suggestion that Instant would be associated with a place or time zone.

Reply:
Good idea. Removed.